tests/issues/issue-1419.php

```diff
@@ -302,19 +302,6 @@
         'standard' => true,
         'full' => true,
     ],
-    'issue-1419-G (lite)' => [
```

Comment:
I removed any lite or standard tests that were still detecting as "Android Webview".  The device-less patterns I added to the 1.0/1.5/1.6 require that "samsung" be present in the device, which doesn't happen for all UAs, so if the device isn't available in the Lite or Standard file, it won't detect them as Samsung Browser, but I've tried to not have mis-matching browser names in tests.